src/engraving/dom/accidental.cpp

```diff
@@ -250,9 +250,9 @@ Accidental::Accidental(EngravingItem* parent)
 //   subTypeUserName
 //---------------------------------------------------------
 
-TranslatableString Accidental::subtypeUserName() const
+TranslatableString Accidental::subtypeUserName() const override
```

Comment:
I beliebe that `override` needs to fo to the declaration, in the header file, nit to the definition in the cpp file.